pygfx/renderers/wgpu/meshshader.py

```diff
@@ -236,8 +269,18 @@ def code_vertex(self):
 
             // Select what face we're at
             let index = i32(in.vertex_index);
-            let face_index = index / 3;
-            var sub_index = index % 3;
+            let face_index = index / {{indexer}};
+            var sub_index = index % {{indexer}};
+            var face_sub_index = 0;
+
+            // for quads assuming the vertices are oriented, the triangles are 0 1 2 and 0 2 3
+            $$ if indexer == 6
+                var quad_map = array<i32,6>(0, 1, 2, 0, 2, 3);
+                //face_sub_index returns 0 or 1 in picking for quads. So the triangle of the quad can be identified.
+                var face_map = array<i32,6>(0, 0, 0, 1, 1, 1);
+                face_sub_index = face_map[sub_index];
+                sub_index = quad_map[sub_index];
+            $$ endif
```

Comment:
The sub_index tweaks to fix the winding just below also needs to be adjusted. If I flip the camera scale, I get nice art!

<img width="533" alt="image" src="https://github.com/pygfx/pygfx/assets/3015475/456f4f6b-d855-4063-82ad-8c856709f669">


Looking into it.